bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/provider/ShellyChannelDefinitions.java

```diff
@@ -519,7 +519,7 @@ public static Map<String, Channel> createSensorChannels(final Thing thing, final
                     CHANNEL_SENSOR_VIBRATION);
         }
         // Create tilt for DW/DW2, for BLU DW create channel even tilt is currently not reported
-        if (sdata.accel != null || (profile.isBlu && sdata.lux != null)) {
+        if (sdata.accel != null || (profile.isBlu && profile.isDW && sdata.lux != null)) {
```

Comment:
Does it still match the comment above "for BLU DW create channel **even tilt is currently not reported**"? Does `profile.isDW` mean that it is reported?